collector/scraper.go

```diff
@@ -261,11 +262,11 @@ func (s *Scraper) scrapeTargets(ctx context.Context) {
 			wr = s.flushBatchIfNecessary(ctx, wr)
 		}
 		if err := iter.Err(); err != nil {
-			logger.Errorf("Failed to scrape %s: %s", target.Addr, err.Error())
+			logger.Warnf("Failed to scrape %s/%s/%s at %s: %s", target.Namespace, target.Pod, target.Container, target.Addr, err.Error())
```

Comment:
the log line here clashes with line220. May be a slight reword will help clarify where the error is coming from.

Reply:
Updated